perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -189,9 +189,19 @@ sub load_problem_or_display_error : Private {
         my $from_email = $c->sessionid && $c->flash->{alert_to_reporter} && $c->flash->{alert_to_reporter} == $problem->id;
 
         my $allowed = 0;
-        $allowed = 1 if $from_email;
-        $allowed = 1 if $c->user_exists && $c->user->id == $problem->user->id;
-        $allowed = 1 if $permissions->{report_inspect} || $permissions->{report_mark_private};
+        if ( $problem->category eq 'Bulky cancel' ) {
```

Comment:
Making this change in the core FMS code doesn't feel quite right somehow, but I'm struggling to think of an alternative... Does the user who made the original booking ever actually see the cancellation ID or a link to the report in question at all? If not, then there's presumably no way they'd end up on this page?

Reply:
Hmm it is actually shown on their 'Your Account' page
<img width="696" alt="Screenshot 2023-01-06 at 09 41 30" src="https://user-images.githubusercontent.com/11098355/210974748-50a724e8-e0cf-4dba-aef6-ac4e584122fc.png">

but if they click through they get an 'Access Denied' message.
Would it be better to hide the link there, rather than have the check I've added here?